include/seqan3/io/alignment_file/output_format_concept.hpp

```diff
@@ -145,6 +149,7 @@ SEQAN3_CONCEPT alignment_file_output_format =
  * \param[in]     ref_id     The data for seqan3::field::REF_ID, e.g. the reference id..
  * \param[in]     ref_offset The data for seqan3::field::REF_OFFSET, i.e. the start position of the alignment in \p ref_seq.
  * \param[in]     align      The data for seqan3::field::ALIGN, e.g. the alignment between query and ref.
+ * \param[in]     cigar_vector The data for seqan3::field::CIGAR, e.g. representing the alignment between query and ref.
```

Comment:
you might want to align all parameters. Or just remove it completely for exact this reason 😄